### DISCUSSION TITLE
Get subtitles from a video.

### DISCUSSION THREAD

**Discussion by CarimboDev** [SubtitleEdit/subtitleedit#5885] [Ideas]
I wanted to know if there is a way to add a way to select the language of the video and get the subtitles in srt format as it works on this site: https://www.subtitlevideo.com/

I tried to get the subtitles normally in subtitle edit but it didn't detect the subtitles.

**Comment by niksedk** [comment]
Could you explain more?

You want the subtitles from the audio or from hard-burned-sub?

I presume it's from the audio... check Video - Video/audio to text...

**Comment by CarimboDev** [comment]
I want hard-burn-sub subtitles

**Reply by niksedk** [reply]
Sorry, SE does cannot do this.
